src/main/java/com/github/vaibhavsinha/kong/api/admin/PluginService.java

```diff
@@ -23,6 +22,9 @@
 
     Plugin updatePlugin(String nameOrId, Plugin request);
 
+    /**
+     * @deprecated use {@link #addPlugin(Plugin)} or {@link #updatePlugin(String, Plugin)}
+     */
     @Deprecated
     Plugin createOrUpdatePlugin(Plugin request);
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)